rel/config/Makefile

```diff
@@ -19,3 +19,7 @@ release: ## Build the OTP releases
 		cd build_vms/ubuntu && \
 		vagrant up && \
 		vagrant halt
+
+.PHONY: dev
+dev:
+	iex -S mix
```

Comment:
I hesitated to leave it there, but if you want to update the way the dev env is started (with a dev.exs file per instance), then there is one entry point to update